.github/workflows/backend-ci.yml

```
@@ -15,6 +15,7 @@ on:
 env:
   CARGO_TERM_COLOR: always
   RUST_BACKTRACE: 1
+  DATABASE_URL: sqlite::memory:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 18: mapping values are not allowed in this context

(syntax-check)